apps/web/src/common/modules/widgets/_components/WidgetFormDataTableCardTransformPivotForm.vue

```diff
@@ -0,0 +1,261 @@
+<script setup lang="ts">
+import {
+    computed, reactive, watch,
+} from 'vue';
+
+import {
+    PDivider, PFieldGroup, PSelectButton, PTextInput, PSelectDropdown,
+} from '@cloudforet/mirinae';
+import type { MenuItem } from '@cloudforet/mirinae/types/inputs/context-menu/type';
+
+import { i18n } from '@/translations';
+
+import { useProxyValue } from '@/common/composables/proxy-state';
+import WidgetFormDataTableCardTransformDataTableDropdown
+    from '@/common/modules/widgets/_components/WidgetFormDataTableCardTransformDataTableDropdown.vue';
+import WidgetFormDataTableCardTransformFormWrapper
+    from '@/common/modules/widgets/_components/WidgetFormDataTableCardTransformFormWrapper.vue';
+import {
+    type DATA_TABLE_OPERATOR, DEFAULT_TRANSFORM_DATA_TABLE_VALUE_MAP,
+} from '@/common/modules/widgets/_constants/data-table-constant';
+import { useWidgetGenerateStore } from '@/common/modules/widgets/_store/widget-generate-store';
+import type { TransformDataTableInfo } from '@/common/modules/widgets/types/widget-data-table-type';
+import type { PivotOptions } from '@/common/modules/widgets/types/widget-model';
+
+
+
+interface Props {
+    dataTableId: string;
+    dataTableInfo: TransformDataTableInfo;
+    formData: Omit<PivotOptions, 'dataTableId'>;
+}
+
+const props = defineProps<Props>();
+
+const emit = defineEmits<{(e: 'update:form-data', value: Omit<PivotOptions, 'dataTableId'>): void;
+    (e: 'update:data-table-info', value: TransformDataTableInfo): void;
+}>();
+const widgetGenerateStore = useWidgetGenerateStore();
+const widgetGenerateState = widgetGenerateStore.state;
+
+const storeState = reactive({
+    dataTables: computed(() => widgetGenerateState.dataTables),
+});
+const state = reactive({
+    isInitiated: true,
+    proxyDataTableInfo: useProxyValue<TransformDataTableInfo>('dataTableInfo', props, emit),
+    proxyFormData: useProxyValue<Omit<PivotOptions, 'dataTableId'>>('formData', props, emit),
+    selectedDataTable: computed(() => storeState.dataTables.find((dataTable) => dataTable.data_table_id === state.proxyDataTableInfo.dataTableId)),
+    labelFieldItems: computed<MenuItem[]>(() => {
+        if (!state.selectedDataTable) return [];
+        const labelsInfo = state.selectedDataTable.labels_info;
+        return Object.keys(labelsInfo).map((key) => ({
+            name: key,
+            label: key,
+        }));
+    }),
+    dataFieldItems: computed<MenuItem[]>(() => {
+        if (!state.selectedDataTable) return [];
+        const dataInfo = state.selectedDataTable.data_info;
+        return Object.keys(dataInfo).map((key) => ({
+            name: key,
+            label: key,
+        }));
+    }),
+    columnFieldInvalid: computed<boolean>(() => {
+        if (state.labelFieldItems.length === 1) return true;
+        return false;
+    }),
```

Comment:
question: Why didn't u use fieldValidator or formValidator? It has different behavior?